rust/kernel/src/printk.rs

```diff
@@ -6,13 +6,18 @@ use core::fmt;
 use crate::bindings;
 use crate::c_types::c_int;
 
+pub use crate::bindings::{
+    KERN_ALERT, KERN_CRIT, KERN_DEBUG, KERN_EMERG, KERN_ERR, KERN_INFO, KERN_NOTICE, KERN_WARNING,
+};
```

Comment:
We should not just re-export these as the public API. We should have a type-safe API for specifying these if they're aprt of the public API.

Reply:
Ok, so a level enum

Reply:
Yeah, that'd be good.